src/components/admin/ImportDialog.tsx

```
@@ -0,0 +1,187 @@
+      }
+
+      // Check if first line is a header
+      const firstLine = nonEmptyLines[0].toLowerCase().trim();
+      const isHeader = firstLine.includes('email') || firstLine.split(',')[0].trim() === 'email';
+      
+      // Process the lines, skipping header if present
+      const startIndex = isHeader ? 1 : 0;
+      const parsedEmails = nonEmptyLines
+        .slice(startIndex)
+        .map(line => line.trim())
+        .filter(line => line && line.includes('@'))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)